### PR TITLE
End the project

### PR DESCRIPTION
**Pull request by Metsanpeitto**
# Project completed 

**I realize just when ending the project that I didn't commit the changes until the end. In the future, I will pay more attention and commit every time I do changes in my projects.**

** On the other hand, the project follows all the guidelines and has not Linters error or bad practices. 
 I request you please review the code and hope it is ok for you as well**

I wait hesitantly for your approval to finally merge the branch  🚀 

### 💻 👩🏽‍💻  Happy Coding Review!  👨🏽‍💻 🖥️

------

### CODE REVIEW

**Comment by Metsanpeitto** [comment]
# Project completed ( I hope !)

**After pushing and requesting a pull I did have to go back and fix many issues that Linters pointed to me.
In the process, I did realize many commits and comment on them in the best way I know. 
I did follow best practices and paid attention to details, so now I request that you please, kindly review my code and let me know if there is something that needs to be improved.**

I wait hesitantly for your approval to finally merge the branch  🚀 

### 💻 👩🏽‍💻  Happy Coding Review!  👨🏽‍💻 🖥️

------

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+- HTML/CSS
+- Node.JS
+
+## Getting Started
+
+If you want a copy of this file go to the github repository and download it
+
+`git clone https://github.com/Metsanpeitto/Portfolio_1`
+
+
```
Comment (Dipeshtwis) [inline README.md]: - [ ] Also mention after clone which file needs to run. In this case `index.html`.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link href="./style.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header>
+      <h3 class="logo-text">My Logo</h3>
```
Comment (Dipeshtwis) [inline index.html]: - [ ] The logo should have a link (`a` tag).

File: index.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+      <img
+        class="menu"
+        src="./images/menu.svg"
+        alt="triangle with all three sides equal"
+        height="87"
+        width="100"
+      />
```
Comment (Dipeshtwis) [inline index.html]: - [ ] The hamburger button should have a `nav` tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+            height="20"
+            width="20"
+          />
+          <img
+            src="./images/Vector.svg"
+            class="icon"
+            alt="bird"
+            height="20"
+            width="20"
+          />
```
Comment (Dipeshtwis) [inline index.html]: - [ ] The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (a tag)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+        alt="bird"
+        height="11"
+        width="10"
+      />
+      <img
+        src="./images/Rectangle-7.svg"
+        class="r-7"
+        alt="bird"
+        height="11"
+        width="10"
```
Comment (Dipeshtwis) [inline index.html]: - [ ] This section has the proper background image (decorations should be with background images, not `img` tags). As Background image helps to write the text over them as well.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+      <img
+        class="menu"
+        src="./images/menu.svg"
+        alt="triangle with all three sides equal"
+        height="87"
+        width="100"
+      />
+    </header>
+    <main>
+      <h1 class="header-text">I'm Abraham Rodriguez Martinez to see you!</h1>
```
Comment (Dipeshtwis) [inline index.html]: - [ ] This texts are overlapping with the paragraph. Please fix that, 
- [ ] Also stick to the design for now I am seeing the website doesn't cover the whole page. Please fix that.

 Below is the screenshot.


Screenshot from 2021-06-03 19-45-51

File: style.css [hunk 7]
```diff
@@ -0,0 +1,197 @@
...
+.menu:hover {
+  transform: scale(1.2);
+}
+
+main {
+  width: 375px;
+  height: 604px;
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
```
Comment (Dipeshtwis) [inline style.css]: - [ ] You have used background property here, But there is no background image. Please include that or remove these unnecessary code.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Metsanpeitto, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: Good use of Flexbox.
:+1: README file is looking good

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Dipeshtwis** [comment]
> # Project completed ( I hope !)
> **After pushing and requesting a pull I did have to go back and fix many issues that Linters pointed to me. In the process, I did realize many commits and comment on them in the best way I know. I did follow best practices and paid attention to details, so now I request that you please, kindly review my code and let me know if there is something that needs to be improved.**
> 
> I wait hesitantly for your approval to finally merge the branch 
> 
> ###  👩🏽‍💻 Happy Coding Review! 👨🏽‍💻

It's good that you realized and also commit for not making same mistake. :+1:

**Review by Shaher-11 — APPROVED** [review]
Hello @Metsanpeitto 

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 

## Optional Suggestions 💡
_The following point is optional to implement however it's highly recommended!_:blush:

- It is better to have the screenshot in the Readme file of the actual mobile view.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**